sc2/pixel_map.py

```diff
@@ -73,11 +73,7 @@ def flood_fill(self, start_point: Point2, pred: Callable[[int], bool]) -> Set[Po
 
             if pred(self[x, y]):
                 nodes.add(Point2((x, y)))
-                for a in [-1, 0, 1]:
-                    for b in [-1, 0, 1]:
-                        if not (a == 0 and b == 0):
-                            queue.append(Point2((x + a, y + b)))
-
+                queue += [Point2((x + a, y + b)) for a in [-1, 0, 1] for b in [-1, 0, 1] if not (a == 0 and b == 0)]
```

Comment:
IMO This makes it harder to read

Reply:
In my opinion you can see immediately that before and after are doing the same thing. The list comprehension can be read as pseudocode.

Reply:
Fair enough